z2/src/deployer.rs

```diff
@@ -522,7 +522,9 @@ pub async fn run_restore(config_file: &str, filename: &str, max_parallel: usize)
 }
 
 pub async fn run_reset(config_file: &str, node_selection: bool) -> Result<()> {
-    let config = NetworkConfig::from_file(config_file).await?;
+    let config = NetworkConfig::from_file(config_file)
+        .await
+        .context(format!("Cannot read {config_file}"))?;
```

Comment:
having the function `from_file` used in multiple places, may be worth adding `context` within it rather than having different behaviours?

Reply:
Will do this in a separate PR else this one will never merge..